### PR TITLE
Re-design #2

### PR DESCRIPTION
**Pull request by eronisko**

### CODE REVIEW

File: app.vue [hunk 1]
```diff
@@ -57,59 +92,75 @@ function onMicrioError() {
...
+          enter-to-class="opacity-100"
+          enter-active-class="transition-all duration-200 ease-out"
+          leave-from-class="opacity-100"
+          leave-to-class="opacity-0"
+          leave-active-class="transition-all duration-100 ease-in"
+        >
+          <div
+            v-if="micrio?.tour && micrio?.marker"
+            class="pointer-events-none absolute inset-0 flex p-16"
+            :class="
```
Comment (FrantisekMichalSebestyen) [inline app.vue:129]: Might be worth extracting to functin here

**Review by FrantisekMichalSebestyen — CHANGES_REQUESTED** [review]
Looks good, please just check this bug.
I had the same problem in my PR.
I fixed it by creating "is-marker-visible" ref in Micrio, which wasn't so nice but seemed to work

https://github.com/SlovakNationalGallery/sakra-stex/assets/28804184/6f290543-144a-4594-bd64-d69725ddf5d4

File: components/pulsatingMarkersTimeline.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import gsap from "gsap";
```
Comment (FrantisekMichalSebestyen) [inline components/pulsatingMarkersTimeline.ts:1]: Why is this in a separate file?

**Comment by eronisko** [comment]
@FrantisekMichalSebestyen Could you try again please?

File: app.vue [hunk 3]
```diff
@@ -59,12 +59,12 @@ watch(micrio, (micrio, oldMicrio) => {
...
-  if (micrio.value.marker.id === marker.id) {
-    micrio.value.tour!.cancel();
-  }
-}
+// function onMarkerClick(marker: Micrio["Marker"]) {
+//   if (!micrio.value?.marker?.id) return;
+//   if (micrio.value.marker.id === marker.id) {
+//     micrio.value.tour!.cancel();
+//   }
+// }
```
Comment (FrantisekMichalSebestyen) [inline app.vue:67]: del

**Review by FrantisekMichalSebestyen — APPROVED** [review]
just del unused fn, then its good

**Pull request merged**